packages/gestalt/src/utils/positioningUtils.js

```diff
@@ -160,35 +160,41 @@ export function getPopoverDir({
   const skipNoEdgeCondition =
     isScrollBoundaryContainer && (nonTopEdge || nonBottomEdge) && (nonLeftEdge || nonRightEdge);
 
-  if (!skipNoEdgeCondition && (nonTopEdge || nonBottomEdge)) {
+  const maybeEdgeCondition = idealDirection !== 'forceDown' && !skipNoEdgeCondition;
+
+  if (maybeEdgeCondition && (nonTopEdge || nonBottomEdge)) {
     left = 0;
     right = 0;
   }
 
   // LEFT or RIGHT: trigger is too close to the left/right of screen for up & down popovers\
-  if (!skipNoEdgeCondition && (nonLeftEdge || nonRightEdge)) {
+  if (maybeEdgeCondition && (nonLeftEdge || nonRightEdge)) {
     up = 0;
     down = 0;
   }
 
   // Choose the main direction for the popover based on available spaces & user preference
   const spaces = [up, right, down, left];
 
-  let popoverDir;
+  if (idealDirection && idealDirection === 'forceDown') {
```

Comment:
If you just put this at the top, do you need the `maybeEdgeCondition` case at all